flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -164,6 +166,11 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler {
     result.success(newDebugImages)
   }
 
+  private fun closeNativeSdk(call: MethodCall, result: Result) {
```

Comment:
```suggestion
  private fun closeNativeSdk(result: Result) {
```

apparently we don't need it